config/docker_etl.config.src

```diff
@@ -0,0 +1,16 @@
+[
+ "config/sys.config",
+ {lager,
```

Comment:
Almost there.. the blockchain stanza has `data`  for the blockchain folder which you will need to override in this file to `/var/data` right?

Reply:
Good catch! Glad you are good reviewer as I am just stabbing in the dark.